### PR TITLE
Including asset_type filter for metrics API

### PR DESCRIPTION
**Pull request by supritat**
### Requirements

* Metrics should be sent based on asset_type filter

### Description of the Change

Updated metrics.json with metricName and dimensions for other assets.

### Benefits

Metrics will be fetched for individual asset of respective service.

### CODE REVIEW

File: core/jazz_metrics/components/metrics.json [hunk 1]
```diff
@@ -123,132 +166,298 @@
...
+            "Unit": "Milliseconds"
+          },
+          {
+            "Label": "Request Processing Latency",
+            "MetricName": "request_processing_latency",
+            "Statistics": "Min",
+            "Unit": "Milliseconds"
+          }
+        ],
+        "dimensions": ["apiproxy"]
```
Comment (raghits) [inline core/jazz_metrics/components/metrics.json]: Can we fix these whitespace changes?

Comment (supritat) [reply core/jazz_metrics/components/metrics.json]: After applying beautify Whitespace got added.

Comment (supritat) [reply core/jazz_metrics/components/metrics.json]: After applying beautify Whitespace got added.

Comment (raghits) [reply core/jazz_metrics/components/metrics.json]: But that just messes the code annotation/blame history. If it is not too much effort, can u remove those manually? Maybe Sini had similar issues.

Comment (supritat) [reply core/jazz_metrics/components/metrics.json]: Done

**Pull request merged**